addons/hw_drivers/tools/helpers.py

```diff
@@ -188,6 +191,22 @@ def save_conf_server(url, token, db_uuid, enterprise_code):
     write_file('odoo-db-uuid.conf', db_uuid or '')
     write_file('odoo-enterprise-code.conf', enterprise_code or '')
 
+def generate_password():
+    """
+    Generate an unique code to secure raspberry pi
+    """
+    alphabet = string.ascii_letters + string.digits
```

Comment:
maybe same as here and remove il1o0
https://github.com/odoo/odoo/blob/master/addons/web/static/src/public/database_manager.js#L38